src/Components/OpenInExploreLogsButton/OpenInExploreLogsButton.test.tsx

```diff
@@ -34,7 +34,7 @@ describe('OpenInExploreLogsButton', () => {
     expect(linkButton).toBeInTheDocument();
     expect(linkButton).toHaveAttribute(
       'href',
-      'http://localhost:3000/a/grafana-lokiexplore-app/explore/job/test-job/logs?var-datasource=test-datasource&from=now-1h&to=now&var-filters=job%7C%3D%7Ctest-job'
+      'http://localhost:3000/a/grafana-lokiexplore-app/explore/job/test-job/logs?var-datasource=test-datasource&from=now-1h&to=now&var-filters=job%7C%3D%7C__CV%CE%A9__test-job%2Ctest-job'
```

Comment:
If you want to export the `encodeFilter` method from links.test.ts it might help make this a bit easier to read/maintain